packages/ketcher-react/src/script/ui/views/modal/components/document/Save/Save.module.less

```diff
@@ -43,7 +43,7 @@
     height: fit-content;
```

Comment:
As of now, even 1 line of warnings causes overflow.
![image](https://user-images.githubusercontent.com/18420141/156412610-c7e817d6-954a-45d5-a857-2912454e70c1.png)

I suggest that we avoid unnecessary overflow and scroll when we're within `50px` of max-height by using this code or something like it:
```diff
.warnings {
  max-height: 50px;
- height: fit-content;
+ flex-shrink: 0;
  border: none;
  border-radius: 4px;
  margin: 2px 0;
  padding: 2px 5px;
  overflow: auto;
```